projects/fio/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build project
+cd fio
```

Comment:
This is wrong, remove "cd fio" AND set "WORKDIR $SRC/fido" in Dockerfile. Otherwise this breaks local reproduction.

Reply:
Ok.
I did not know about this.
Is this about https://google.github.io/oss-fuzz/advanced-topics/reproducing/#reproduce-using-local-source-checkout ?